### DISCUSSION TITLE
actions/checkout@v4 can't find .editorconfig file.May I ask how to solve this problem?

### DISCUSSION THREAD

**Discussion by hanjiezhou** [community/community#129519] [Actions]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/101302973/08c017b8-07c6-4464-8721-2f5a1ad00cd4)

**Comment by manav-gopal** [comment]


**Reply by hanjiezhou** [reply]
But this section is actions/ checkout@v4  that have already been integrated .I can only use it but cannot modify it.
